pkg/storage/storage.go

```diff
@@ -668,3 +679,71 @@ func (s *storageModule) Monitor(ctx context.Context) <-chan pkg.PoolsStats {
 
 	return ch
 }
+
+func (s *storageModule) periodicallyCheckDiskShutdown() {
+	ticker := time.NewTicker(5 * time.Minute)
+	quit := make(chan struct{})
+	go func() {
+		for {
+			select {
+			case <-ticker.C:
+				log.Info().Msg("Checking pools for disks that should be shutdown...")
+				s.shutdownDisks()
+			case <-quit:
+				ticker.Stop()
+				return
+			}
+		}
+	}()
+}
+
+// shutdownDisks will check the disks power status.
+// If a disk is on and it is not mounted then it is not supposed to be on, turn it off
+func (s *storageModule) shutdownDisks() {
+	for _, pool := range s.pools {
+		for _, device := range pool.Devices() {
+			log.Debug().Msgf("checking device: %s", device.Path)
+			on, err := checkDiskPowerStatus(device.Path)
+			if err != nil {
+				if err, ok := err.(*exec.ExitError); ok {
+					log.Err(err).Msgf("skipping device")
+					// if cmd exits with exit error the device is shutdown
+					continue
+				}
+			}
+
+			_, mounted := pool.Mounted()
+			if mounted {
+				continue
+			}
+
+			if on {
+				log.Debug().Msgf("shutting down device %s because it is not mounted and the device is on", device.Path)
+				err := pool.Shutdown()
+				if err != nil {
+					log.Err(err).Msgf("failed to shutdown device %s", device.Path)
+					return
+				}
+			}
+		}
+	}
+}
+
+func checkDiskPowerStatus(path string) (bool, error) {
+	output, err := exec.Command("smartctl", "-i", "-n", "standby", path).Output()
+	if err != nil {
```

Comment:
you should instead do the check here and just return the proper "bool"/error flag
if the cmd exits with non success error if the device is down, you should do 
```
if (err is exit error) {
   return false, nil
} else {
   return false, err
}
```
for example